app/src/androidTest/java/ch/epfl/sdp/drone3d/service/storage/FirebaseDatabaseTest.kt

```diff
@@ -42,42 +39,38 @@ class FirebaseDatabaseTest {
     fun loadUserPseudoReturnsExpectedValues() {
         val db = FirebaseDatabase()
         val expectedPseudo = "Masachouette"
-        val UID = "123456"
+        val uid = "123456"
         val counter = CountDownLatch(1)
 
-        database.getReference("users/$UID/pseudo")
+        database.getReference("users/$uid/pseudo")
             .setValue(expectedPseudo)
 
-        val data = db.loadUserPseudo(UID)
+        val data = db.loadUserPseudo(uid)
         data.observeForever { pseudo ->
-            assertThat(pseudo, CoreMatchers.equalTo(expectedPseudo))
+            assertThat(pseudo, equalTo(expectedPseudo))
             counter.countDown()
         }
 
 
         counter.await(timeout, TimeUnit.SECONDS)
-        assertThat(counter.count, CoreMatchers.equalTo(0L))
+        assertThat(counter.count, equalTo(0L))
     }
 
     @Test
     fun storeUserPseudoCreateExpectedValueWhenNoPreviousPseudo() {
         val db = FirebaseDatabase()
         val expectedPseudo = "Masachouette"
-        val UID = "123456"
+        val uid = "123456"
         lateinit var pseudo: String
         val counter = CountDownLatch(1)
 
-
         val pseudoListener = object : ChildEventListener {
-            override fun onCancelled(error: DatabaseError) {
-                TODO("Not yet implemented")
-            }
-            override fun onChildMoved(snapshot: DataSnapshot, previousChildName: String?) {
-                TODO("Not yet implemented")
-            }
-            override fun onChildChanged(snapshot: DataSnapshot, previousChildName: String?) {
-                TODO("Not yet implemented")
-            }
```

Comment:
Why did you remove the TODO ?

Reply:
As they were created for the tests, those methods will never have more code in it.